gamemodes/terrortown/gamemode/client/cl_tbuttons.lua

```diff
@@ -210,6 +225,10 @@ function TBHUD:Draw(client)
 		scrpos = focus_but.ent:GetPos():ToScreen()
 		scrPosXMid, scrPosYMid = scrpos.x - mid, scrpos.y - mid
 
+		if (val.access or val.visible) and ent:GetDelay() > 0 and percent > 0 then
```

Comment:
When hovering make the alpha maxed out for completed/black part
```suggestion
		if (val.access or val.visible) and ent:GetDelay() > 0 and percent > 0 then
			draw.Arc(ent:EntIndex(), scrpos.x, scrpos.y, circle_radius, outline_width, 0, 360, 0, ColorAlpha(COLOR_BLACK, 200))
```